.github/workflows/e2e.yml

```diff
@@ -145,7 +145,9 @@ jobs:
           CMAKE_ARGS="-DLLAMA_CUBLAS=on" python3 -m pip install --no-binary llama_cpp_python -c constraints.txt llama_cpp_python
           # needed for --4-bit-quant option to ilab model train
           python3 -m pip install bitsandbytes
+          python3 -m pip install packaging wheel
```

Comment:
Why not add these two along with `bitsandbytes` above?

Reply:
Are you building bitsandbytes from sources? In that case you have to install packaging and wheel first, then run a second `pip install` for bitsandbytes.